src/bot.py

```diff
@@ -1,26 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+@author: AlwaysLivid
+@description: Bot that posts a tiny flower bed on Twitter every few hours.
+'''
 
 import tweepy
 import time
 import logging, os, textwrap
 from config import *
 from random import randint
-import sys
-import credentials
+
 print("""
                        TinyFlowerBeds
-
             Copyright (C) 2018-2019 AlwaysLivid
-
 =============================================================
 ======================= DISCLAIMER ==========================
 =============================================================
 This program comes with ABSOLUTELY NO WARRANTY.
 This is free software, and you are welcome to redistribute it
 under certain conditions; read the LICENSE file for details.
 =============================================================
-
 """)
 
+FLAG = True

```

Comment:
What's the point of the flag? It's not explained and not used practically anywhere.

Reply:
Apologies. I was experimenting with exception, nothing to do with actual code.